admin-dev/themes/new-theme/template/components/layout/shop_list.tpl

```diff
@@ -22,29 +22,36 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *}
-{if isset($is_multishop) && $is_multishop && $shop_list &&
-  (isset($multishop_context) &&
-  $multishop_context & Shop::CONTEXT_GROUP ||
-  $multishop_context & Shop::CONTEXT_SHOP ||
-  $multishop_context & Shop::CONTEXT_ALL
-)}
+{if isset($is_multishop)
+  && $is_multishop
+  && $shop_list
+  && (
```

Comment:
Something is weird here, maybe use parenthesis 
```
&& (
    isset($multishop_context)
    && 
    (
        $multishop_context & Shop::CONTEXT_GROUP
        || $multishop_context & Shop::CONTEXT_SHOP
        || $multishop_context & Shop::CONTEXT_ALL
    )
  )}
```

Reply:
Actually I don't think this thing works as intended, but it's not the objective of this PR. I tried to, though.

Reply:
In fact I don't understand what the if must do with mixed && and || :/

Reply:
Honestly... neither do I! We'll figure that out later